api/src/opentrons/protocol_api/module_contexts.py

```diff
@@ -974,3 +974,49 @@ def deactivate_shaker(self) -> None:
     def deactivate_heater(self) -> None:
         """Stop heating."""
         self._module.deactivate_heater()
+
+    def flag_unsafe_move(
+        self,
+        to_loc: types.Location,
+        is_multichannel: bool,
+    ) -> None:
+        """
+        Raise an error if attempting to perform a move that's deemed unsafe due to
+        the presence of the heater-shaker.
+        """
```

Comment:
This is not intended to be part of the public Python Protocol API, right?

Let's hide it from docs.opentrons.com:

```suggestion
        """
        Raise an error if attempting to perform a move that's deemed unsafe due to
        the presence of the heater-shaker.
        
        For Opentrons internal use only.
        
        :meta private:
        """
```